pkg/prometheus/agent/operator.go

```diff
@@ -444,6 +473,11 @@ func (c *Operator) addHandlers() {
 		UpdateFunc: c.handleBmonUpdate,
 		DeleteFunc: c.handleBmonDelete,
 	})
+	c.sconInfs.AddEventHandler(cache.ResourceEventHandlerFuncs{
+		AddFunc:    c.handleBmonAdd,
+		UpdateFunc: c.handleBmonUpdate,
+		DeleteFunc: c.handleBmonDelete,
+	})
```

Comment:
Those should be new functions that handle the ScrapeConfigKind instead of ProbeKind

Reply:
Ooooh, completely overlooked that, thanks! 👍

Reply:
Fixed in 3ed9ecb22!